### ISSUE TITLE
Feel free to share your results here

### ISSUE THREAD

**Issue by mikel-brostrom** [deepakcrk/yolov5-crowdhuman#24]
I am opening this just as a place for people to drop their results and discuss how they got to them :)

**Comment by mikel-brostrom** [comment]
Currently training YOlov5m on Crowdhuman + COCO people

![Screenshot from 2022-10-20 11-37-55](https://user-images.githubusercontent.com/18719680/196913997-5536d931-5431-42c8-abdc-03c20a53e251.png)

**Comment by mikel-brostrom** [comment]
AP for Crowhuman trained on: `train1`, `train2`, `train3`; evaluated on: `val`. 0.7243.

**Comment by mikel-brostrom** [comment]
Now training Yolov5l on full body only from Crowdhuman and Crowdhuman + COCO people, imgz 1280

**Comment by deepakcrk** [comment]
@mikel-brostrom My observation is that, adding more data to training may not increase the AP scores on your validation dataset, as the model may be reached the max capacity of the AP that it can get on that particular validation dataset.

**Comment by mikel-brostrom** [comment]
I agree for the case of Yolov5m. It seems that training data volume is not a problem for the crowhuman dataset when using Yolov5m. However, I wonder if this is also the case for Yolov5l or Yolov5x. As with larger model capacity it may benefit from more data.

**Comment by deepakcrk** [comment]
My experiment shows only minor improvements (less than 2%) in both medium and large. 

My experiment was to improve COCO pre-trained model's AP score on COCO validation data, by adding more data to the COCO train dataset. The improvement that I got is very minor, even by doubling the train dataset size.  My observation is that the COCO pre-trained models already achieved the maximum they can get on the COCOval dataset.
